### DISCUSSION TITLE
wire:loading.remove not reverted in case of request error

### DISCUSSION THREAD

**Discussion by lucasromanojf** [livewire/livewire#3728] [7. Bugs (archived)]
Describe the bug
- When a backend error happens, wire:loading.remove is not reverted from the element.

To Reproduce

- Implement the counter example
- Put a wire:loading.remove on + button
- Throw an exception from increment function
- Click the button that throws the exception (+)
- The + button will be hidden, an exception popup will be shown and, after closing it, the + button won't show back, and the user will need reload the page

Expected behavior

- It's expected that wire:loading.remove is reverted not only after request success, but after request errors too

Desktop (please complete the following information):

- OS: Windows 10 64 Bit
- Browser: Opera 78.0.4093.184 (64 bits)

**Comment by joshhanley** [comment]
@lucasromanojf thanks for reporting! Any chance you could setup an example on https://laravelplayground.com/#/

Thanks!

**Reply by lucasromanojf** [reply]
Hey @joshhanley, sure, gonna do it next week, thanks

**Reply by joshhanley** [reply]
Ok thanks 🙂

**Reply by lucasromanojf** [reply]
Hey @joshhanley , sorry for the delay. Weirdly, it works as expected on Laravel Playground. I'm not sure which version of Livewire is there. I'm using dev-master locally, with the same source code, and the issue happens.

**Reply by joshhanley** [reply]
@lucasromanojf hmm yeah I think Laravel playground is running an old version of Livewire, possibly 2.5.5.

If you already have a small example repo which reproduces the issue with as little code as possible, any chance you could share that, and maybe test on 2.5.5 and 2.6 and 2.7 and see if it's something that has changed in a recent version (if it works on playground)?

Thanks!

**Reply by lucasromanojf** [reply]
Sure @joshhanley gonna set up a small project that reproduces this behavior, thanks!

**Reply by joshhanley** [reply]
Great, thanks! 🙂

**Reply by lucasromanojf** [reply]
Here it is @joshhanley:

https://github.com/lucasromanojf/lwtest

Just visit "/" route, click on the "Show message" button and you will see the error happening. I tried with 2.4, 2.5.5, 2.6 and 2.7, and it happens in all of them.

**Reply by lucasromanojf** [reply]
Hello @joshhanley , I think I was able to PR this fix: #4185

**Reply by joshhanley** [reply]
@lucasromanojf nice! Thanks for that 🙂

**Reply by lucasromanojf** [reply]
PR #4185 merged!

**Comment by lucasromanojf** [comment]
Hello! This error is back on Livewire version 3.
